definitions/types/azure-security-iam.json

```diff
@@ -5,7 +5,7 @@
   "description": "IAM Roles And Scopes",
   "additionalProperties": false,
   "patternProperties": {
-    "^[a-z\/-]+$": {
+    "^[a-z]+[a-z_]*[a-z]$": {
```

Comment:
This will break the dynamodb table bundle. I wouldn't merge that until we change the policy emitted from that artifact.

Reply:
It would also break the `azure-storage-account-application-assets` bundle. But it only breaks them if we re-publish those bundles without changing their artifacts ya?

I was thinking

- update the artifact-definition
- publish the artifact-definition
- make the change in the bundle
- publish the bundle (this _also_ pulls in the new definition)

LMK if there's a better way. I can Slab the process as well 🤙

Reply:
sounds good lets get the PRs good to go and then roll with this.

Reply:
I like that plan. I'll queue them up, link them here, then merge away.